RPGManager_development/src/rpg_database/character_sheet/Armors.java

```diff
@@ -0,0 +1,48 @@
+package rpg_database.character_sheet;

```

Comment:
Annyi, hogy hiányolom a CharacterSheetUnitTests.java-ban a Strainhez tartozó alap teszteket. Normál esetben primitív Fieldekre is tesztelünk egy malformed inputot meg egy set-elést. Mint ahogy egy XPnél vagy egy bármilyen másik alap típusnál tennénk.